src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/providers/GithubApiClientAuthHelper.kt

```diff
@@ -36,18 +34,22 @@ class GithubApiClientAuthHelper(private val appConfig: Config,
         user = getGitHubUser()
     }
 
-    override fun checkGithubUserHasOrderlyWebAccess()
+    override fun checkGitHubOrgAndTeamMembership()
     {
         checkAuthenticated()
 
         val githubOrg = appConfig["auth.github_org"]
         val teamName = appConfig["auth.github_team"]
 
-        if (!githubOrg.isEmpty() && !currentUserBelongsToOrg(githubOrg))
+        if (githubOrg.isEmpty())
+        {
```

Comment:
I've also changed this behaviour to require a GitHub org if auth.provider=github. This is also reflected in the deploy tool.